src/legacy/ui/public/vis/editors/default/components/default_editor_agg_select.tsx

```diff
@@ -80,24 +77,39 @@ function DefaultEditorAggSelect({
       <FormattedMessage
         id="common.ui.vis.defaultEditor.aggSelect.helpLinkLabel"
         defaultMessage="{aggTitle} help"
-        values={{ aggTitle: isAggTypeDefined ? value.title : '' }}
+        values={{ aggTitle: value ? value.title : '' }}
       />
     </EuiLink>
   );
 
+  const errors = [];
+
+  if (!aggTypeOptions.length) {
+    errors.push(
+      i18n.translate('common.ui.vis.defaultEditor.aggSelect.noCompatibleAggsDescription', {
+        defaultMessage:
+          'The {indexPatternTitle} index pattern does not contain any of aggregations.',
```

Comment:
```suggestion
          'The {indexPatternTitle} index pattern does not contain any aggregations.',
```

Reply:
Also, we should probably get @gchaps to review all of the `defaultMessage`s.

Reply:
Same comment as above.  The index pattern {indexPatternTitle} does not contain any of aggregations.